TramsDataApi.Test/UseCases/SearchAcademyConversionProjectsTests.cs

```diff
@@ -20,34 +20,31 @@ public SearchAcademyConversionProjectsTests()
         {
             _fixture = new Fixture();
         }
-        
+        private const string Title = "Title_One";
```

Comment:
small thing, but these fields would normally be positioned above the constructor.

Reply:
Done